spec/requests/organizer_can_message_members_spec.rb

```diff
@@ -0,0 +1,46 @@
+describe 'POST /groups/:group_id/notifications' do
+  let!(:users) { create_list(:user, 3) }
+  let!(:group) { create(:group) } 
+  let!(:member1) { create(:membership, group: group, user: users[1]) } 
+  let!(:member2) { create(:membership, group: group, user: users[2]) }
+
+  describe 'POST req with valid credentials' do
+    let(:user_credentials) { users[0].create_new_auth_token }
+    let(:headers) { { HTTP_ACCEPT: 'application/json' }.merge!(user_credentials) }
+    let(:mail_delivery){ ActionMailer::Base.deliveries[0] }
+    
+    before do
+      post "/groups/#{group.id}/notifications", headers: headers
+    end
+
+    it 'renders the receiver email' do
+      expect(mail_delivery.to).to include(users[1].email)
+      expect(mail_delivery.to).to include(users[2].email)
+      expect(mail_delivery.to).not_to include(users[0].email)
```

Comment:
I would NOT have 3 expect statements in a test and I would like you to pay attention to the message in this test. You say `renders the receiver email` but you test to who the message is being sent to.

Reply:
done

Reply:
done